apps/web/src/components/FilterDialog/FilterDialog.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { StoryFn } from "@storybook/react";
+import { faker } from "@faker-js/faker";
+import { action } from "@storybook/addon-actions";
+
+// import { Combobox } from "@gc-digital-talent/forms";
```

Comment:
Is this needed?

Reply:
Very much so! JK, removed in [remove commented import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8617/commits/7ab363cf6fc06b55ca998bc7f02be5d3412d557a)